pkgs/by-name/ei/eigenlayer-cli/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "eigenlayer-cli";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "NethermindEth";
+    repo = "eigenlayer";
+    rev = "v${version}";
+    hash = "sha256-VO56mMkUVeZN+YJ/cxPyZQ5dGvVDirslH4jhV5hpufQ=";
+  };
+
+  vendorHash = "sha256-DDGMprfryWj9Td4PX/j5Fsjm+bGGiDgnCXmm2IhRSMo=";
+
+  ldflags = [ "-s" "-w" ];
+  subPackages = [ "cmd/eigenlayer" ];
+
+  # Can't pass tests due to mockgen generation
+  doCheck = false;
+
+  postInstall = ''
+    mv $out/bin/eigenlayer $out/bin/eigenlayer-cli
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.eigenlayer.xyz/";
+    changelog = "https://github.com/NethermindEth/eigenlayer/releases/tag/v${version}";
```

Comment:
To make it easier, you can use the `${src.rev}` part to use the complete tag.

Reply:
Good point, done